---
# Documentation: https://sourcethemes.com/academic/docs/managing-content/

title: "An Assessment of the Relationship Between Genomic Connectedness and Prediction Accuracy for Fat Composition Traits in Nellore Beef Cattle"
summary: "Genetic connectedness assesses the extent to which estimated breeding values can be fairly compared across management units. Connectedness in genetic evaluation is important if management units differ in their genetic mean. The concept of genetic connectedness in the whole-genome prediction era can be extended to measure the connectedness level between the reference and validation sets. In genomic prediction (GP), several statistical models address additive and non-additive effects parametrically and nonparametrically. When non-additive genetic variation is accounted, it can be used to predict total genetic values, to increase the efficiency of mate allocation procedures as well as crossbreeding or purebred selection schemes. However, the relationship between the estimated level of connectedness and prediction accuracies in the presence of non-additive genetic variation is less well understood and little is known about the impact of non-additive gene action on genomic connectedness measures. Despite the recent achievements in GP, there is still a drastic shortage of non-additive gene action studies in cattle breeds. The objective of this study is to investigate the relationship between genomic connectedness and prediction accuracy from additive and non-additive gene actions for fat composition traits in Nellore cattle. For fatty acid profile, data from 943 Nellore male animals from farms that integrate DeltaGen, CRV PAINT, and Nelore Qualitas breeding programs, the Thematic Project (FAPESP Process: 2009 / 16118-5), and the Young Researcher Project (FAPESP Process: 2011 / 21241-0) will be used. The fatty acid profile was analyzed in Longissimus thoracis samples using gas chromatography, and capillary column of 100m. Animals were genotyped using the BovineHD BeadChip (High-Density Bovine BeadChip) Illumina® with 777,000 SNPs. The data set for fat composition traits encompasses records from 66,496 females and their relatives, totaling 176,069 phenotypic records for growth, carcass, reproductive, and feed efficiency indicator traits. The pedigree contained information from 244,254 animals, born between 1977 and 2016. A total of 8,652 animals were genotyped with the low-density panel (CLARIFIDE® Nellore 2.0). Genotypes were imputed to a panel containing 735,044 markers using the FIMPUTE 2.2 software. We will assess genome-based connectedness across management units by applying prediction error variance of difference and coefficient of determination. The present project provides a unique opportunity to characterize non-additive gene actions associated with fat composition traits in Nellore cattle. Given that connectedness and prediction accuracies have important influences on genomic selection, this project will be of interest to wider community members including academics and industry professionals."
authors: []
tags: []
categories: []
date: 2020-01-20T11:57:57-03:00

# Optional external URL for project (replaces project detail page).
external_link: ""

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ""
  focal_point: ""
  preview_only: false

# Custom links (optional).
#   Uncomment and edit lines below to show custom links.
# links:
# - name: Follow
#   url: https://twitter.com
#   icon_pack: fab
#   icon: twitter

url_code: "https://bv.fapesp.br/en/bolsas/184280//"
url_pdf: ""
url_slides: ""
url_video: ""

# Slides (optional).
#   Associate this project with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
slides: ""
---
