---
# Documentation: https://sourcethemes.com/academic/docs/managing-content/

title: "Comparison Between Haplotype‐based and Individual Snp‐based Genomic Predictions for Beef Fatty Acid Profile in Nelore Cattle"
authors: ["Feitosa F.L.B.", "Pereira A.S.C.", "Amorim, S. T.", "Peripolli E.", "Silva R.M.O.", "Braz C.U.", "Ferrinho A.M.", "Schenkel F.S.", "Brito L.F.", "Espigolan R.", "de Albuquerque L.G.", "Baldi F."]
date: "2019-12-22"
doi: "https://doi.org/10.1111/jbg.12463"

# Schedule page publish date (NOT publication's date).
publishDate: 2020-01-20T11:16:21-03:00

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["2"]

# Publication name and optional abbreviated publication name.
publication: "Journal of Animal Breeding and Genetics"
publication_short: "Journal of Animal Breeding and Genetics"

abstract: "The aim of this study was to evaluate the genomic predictions using the single‐step genomic best linear unbiased predictor (ssGBLUP) method based on SNPs and haplotype markers associated with beef fatty acids (FAs) profile in Nelore cattle. The data set contained records from 963 Nelore bulls finished in feedlot (±90 days) and slaughtered with approximately 24 months of age. Meat samples from the Longissimus dorsi muscle were taken for FAs profile measurement. FAs were quantified by gas chromatography using a SP‐2560 capillary column. Animals were genotyped with the high‐density SNP panel (BovineHD BeadChip assay) containing 777,962 markers. SNPs with a minor allele frequency and a call rate lower than 0.05 and 0.90, respectively, monomorphic, located on sex chromosomes, and with unknown position were removed from the data set. After genomic quality control, a total of 469,981 SNPs and 892 samples were available for subsequent analyses. Missing genotypes were imputed and phased using the FImpute software. Haplotype blocks were defined based on linkage disequilibrium using the Haploview software. The model to estimate variance components and genetic parameters and to predict the genomic values included the random genetic additive effects, fixed effects of the contemporary group and the age at slaughter as a linear covariate. Accuracies using the haplotype‐based approach ranged from 0.07 to 0.31, and those SNP‐based ranged from 0.06 to 0.33. Regression coefficients ranged from 0.07 to 0.74 and from 0.08 to 1.45 using the haplotype‐ and SNP‐based approaches, respectively. Despite the low to moderate accuracies for the genomic values, it is possible to obtain genetic progress trough selection using genomic information based either on SNPs or haplotype markers. The SNP‐based approach allows less biased genomic evaluations, and it is more feasible when taking into account the computational and operational cost underlying the haplotypes inference."

# Summary. An optional shortened abstract.
summary: ""

tags: []
categories: []
featured: false

# Custom links (optional).
#   Uncomment and edit lines below to show custom links.
# links:
# - name: Follow
#   url: https://twitter.com
#   icon_pack: fab
#   icon: twitter

url_pdf: ''
url_code:
url_dataset:
url_poster:
url_project:
url_slides:
url_source:
url_video:

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ""
  focal_point: ""
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides: ""
---
