---
# Documentation: https://sourcethemes.com/academic/docs/managing-content/

title: "Genome Wide Association Study for Beef Fatty Acid Profile Using Haplotypes in Nellore Cattle"
authors: ["Feitosa, F. L. B.", "Pereira, A. S. C.", "Mueller, L.", "Fonseca, P.", "Braz, C.", "Amorim, S. T.", "Espigolan, R.", "Lemos, M. V. A.", "Albuquerque, L. G.", "Schenkel, F.", "Brito, L.", "Stafuzza, N. B.", "Baldi, F."]
date: "2021-01-10"
doi: "https://doi.org/10.1016/j.livsci.2021.104396"

# Schedule page publish date (NOT publication's date).
publishDate: 2021-08-04T11:42:29-04:00

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["2"]

# Publication name and optional abbreviated publication name.
publication: "Livestock Science"
publication_short: "Livestock Science"

abstract: "The objective of this study was to identify genomic regions associated with beef fatty acid (FA) profile in Nellore cattle finished in feedlot, using haplotype based analyzes. Also, the metabolic pathway enrichment analysis was performed to seek for genes with known functions, close to the haplotype blocks, associated with the evaluated traits, providing subsidies for a better understanding of how the genes affect this trait. A total of 963 Nellore bulls with phenotype for FA, were genotyped using the Illumina BovineHD BeadChip. Samples from the Longissimus thoracis muscle were taken for FAs profile measurement by gas chromatography using a SP-2560 capillary column. Haplotype blocks were defined based on the linkage disequilibrium (LD), where pairs of SNPs are in strong LD if the confidence interval is equal to or greater than 95% with Dâ€™ equal to 0.98 and the lower limit is above 0.70. The model used to estimate the haplotypes effects for the sum of beef fatty acid groups included the fixed effects of contemporary group, animal age at slaughter (as linear covariable), haplotypes (as linear regression over the number of haplotype copies: 0, 1 e 2), and the additive genetic effect as random (polygenic effect). Genome-wide association analyzes were performed for each haplotype block (single marker association) by a Restricted Maximum Likelihood model (REML) using ASREML v. 4.3 software. A total of 15, 15, 4, 3, 6, 5, and 10 haplotypes were significantly associated (FDR< 0.10) with monounsaturated fatty acids (MUFA), polyunsaturated fatty acids (PUFA), total amount of saturated fatty acids (SFA), PUFA:SFA ratio, omega 3 (n3), omega 6 (n6), and n6:n3 ratio, respectively. The statistically significant haplotypes harbored 38 genes, among which we highlighted those involved in lipid biosynthesis and metabolism, low-density lipoprotein oxidation, transport of fatty acids and cholesterol, adipocyte functions, biosynthesis of glycoprotein hormones, regulation of serum triglyceride and high-density lipoprotein-cholesterol levels, and adipogenesis. In addition, several novel regions were identified, including a putative quantitative trait locus for lipid metabolism and energy homeostasis. The enrichment analysis by DAVID tool revealed 14 significant (P< 0.05) terms related to fatty acids such as cellular response to retinoic acid, glycerol transport, regulation of Notch signaling pathway, and Ras signaling pathway. These results would contribute to the improvement of nutritional and health value of beef lipids in indicine cattle by including information from causal mutations in genetic evaluation through weighted their effects differently in genomic predictions."

# Summary. An optional shortened abstract.
summary: ""

tags: []
categories: []
featured: false

# Custom links (optional).
#   Uncomment and edit lines below to show custom links.
# links:
# - name: Follow
#   url: https://twitter.com
#   icon_pack: fab
#   icon: twitter

url_pdf: ''
url_code:
url_dataset:
url_poster:
url_project:
url_slides:
url_source:
url_video:

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ""
  focal_point: ""
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides: ""
---
